README.md

```diff
@@ -584,13 +584,11 @@ On Ubuntu using Unix Makefiles
 Most dependencies can be obtained via the Ubuntu software repositories. On each
 line below, we show the corresponding package.
 
-* **operating system**: Ubuntu 13.10 or later.
+* **operating system**: Ubuntu 12.04 or later. *(See OS specific instructions under 'For the impatient' below)*
```

Comment:
What do you think about doing the following:

1. Removing the `**operating system**` bullet from the itemized list of dependencies.
2. Modifying the sentences that come before the list of the dependencies:
  Before:

  > Most dependencies can be obtained via the Ubuntu software repositories. On each line below, we show the corresponding package.

  After:

  > Most dependencies can be obtained via the Ubuntu software repositories; especially if you are using Ubuntu 15.10 or later. On each line below, we show the corresponding package name. You can find instructions for specific Ubuntu versions under 'For the impatient' below.